### DISCUSSION TITLE
Thank you  😍

### DISCUSSION THREAD

**Discussion by aahnik** [mesuutt/read-medium-extension#2] [General]
this is so damn useful. There are some other extensions, most of which dont work and some are not open source. I fear they steal my data.

Your extension is so simple and smart. I dont know js, but i saw your code. It is so simple and smart and the logic makes sense to me easily.

Earlier I used to open in incognito, your extension saves me time.

Thanks again

**Comment by AntonioParroni** [comment]
Yeah. Thank you so much!
